docs/components/modeler/web-modeler/advanced-modeling/camunda-docs-ai.md

```diff
@@ -0,0 +1,22 @@
+---
+id: camunda-docs-ai
+title: Camunda Docs AI
+description: Ask our smart AI-powered chatbot technical and non-technical questions about Camunda 8.
+---
+
+<span class="badge badge--cloud">Camunda 8 only</span>
+<span class="badge badge--alpha">Alpha</span>
+
+:::note
+Camunda Docs AI is an alpha feature. To use this feature, enable the [AI-powered features](https://camunda.com/blog/2024/02/camunda-docs-ai-developer-experience-new-level/) through the [alpha features](/components/console/manage-organization/enable-alpha-features.md) menu. Learn more about [alpha features and general availability](/reference/alpha-features.md).
+:::
+
+Camunda Docs AI provides a smart AI-powered chatbot that helps you find answers to your technical and non-technical questions about Camunda within Web Modeler, rather than navigating and searching across multiple sources of information like documentation, forums, blog posts, etc.
+
+![camunda docs ai widget](./img/camunda-docs-ai-widget.png)
+
+Click the **Camunda Docs AI** widget in the bottom right corner of a BPMN/DMN canvas to open the chatbot:
```

Comment:
To clarify, will this _only_ show up from within a BPMN/DMN canvas in Web Modeler SaaS?

Reply:
Yes, only in BPMN/DMN canvas for now and this is an experimental feature.